plugin/helpers.go

```diff
@@ -61,6 +61,12 @@ type Helpers interface {
 	// Minimum server version: 5.6
 	KVListWithOptions(options ...KVListOption) ([]string, error)
 
+	//KVModify modifies data for a particular key
+	//
+	//@tag KeyValueStore
```

Comment:
```suggestion
	// @tag KeyValueStore
```

@hanzei is it valid putting tags on helper functions? I thought they only go under the plugin API.

Reply:
Tags in helpers are not shown in https://developers.mattermost.com/extend/plugins/server/reference/#Helpers, but we might add it in the future. But I see that it's confusing from someone. Let's drop it.